aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -536,12 +536,16 @@ Resources:
                   - 'rds:RestoreDBClusterToPointInTime'
                   - 'rds:DeleteDBCluster'
                   - 'rds:DescribeDBClusterEndpoints'
-                Resource: !Sub "arn:aws:rds:${AWS::Region}:${AWS::AccountId}:cluster:<%=CDO.db_cluster_id%>*"
+                Resource:
+                  - !Sub "arn:aws:rds:${AWS::Region}:${AWS::AccountId}:cluster:<%=CDO.db_cluster_id%>*"
```

Comment:
I think * matches zero characters, so this would grant permission to delete the actual DB cluster in addition to the clone. To be extra safe, can you include at least a dash character after the db_cluster_id (and maybe the whole clone suffix)? You may also have to break out the DeleteDBCluster action from some of the others in this group, since maybe some of the others require access to both the clone and original.